### PR TITLE
DOC-2016: Added TINY-9749 to the release notes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2016

Changes:
* Added TINY-9749 to the release notes.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by lorenzo-pomili — APPROVED** [review]
This change was specific for when the adcode editor is in modal, maybe it could be useful to mention it in the note

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 1]
```diff
@@ -30,6 +30,21 @@
...
+
+=== Advanced Code Editor did not preserve either the `font size` and `dark mode` save state when closed.
+//#TINY-9749
+
+In previous versions of {productname}, the `font size` and `dark mode` state within the Advanced Code Editor was not preserved after closing the dialog.
+
+As a consequence, when the user closed the Advanced Code Editor the editor would reset the state back to its default settings.
+
+In {productname} 6.5, the state for the `font size` and `dark mode` status is now preserved when the Advanced Code Editor is closed.
+
```
Comment (kemister85) [inline modules/ROOT/pages/6.5-release-notes.adoc:47]:
```suggestion

[NOTE]
This issue only affected the Advanced Coded Editor when it was is in modal.
```

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 2]
```diff
@@ -30,6 +30,23 @@
...
+[NOTE]
+This issue only affected the Advanced Coded Editor when it was is in modal.
 [[accompanying-premium-plugin-end-of-life-announcement]]
```
Comment (kemister85) [inline modules/ROOT/pages/6.5-release-notes.adoc:50]:
```suggestion
This issue only affected the Advanced Coded Editor when it was is in modal.

[[accompanying-premium-plugin-end-of-life-announcement]]
```

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 3]
```diff
@@ -30,6 +30,24 @@
...
+//#TINY-9749
+
+In previous versions of {productname}, the `font size` and `dark mode` state within the Advanced Code Editor was not preserved after closing the dialog.
+
+As a consequence, when the user closed the Advanced Code Editor the editor would reset the state back to its default settings.
+
+In {productname} 6.5, the state for the `font size` and `dark mode` status is now preserved when the Advanced Code Editor is closed.
+
+[NOTE]
+This issue only affected the Advanced Coded Editor when it was is in modal.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:49]:
```suggestion
=== Advanced Code Editor 3.3.0

The {productname} 6.5 release includes an accompanying release of the **Advanced Code Editor** premium plugin.

**Advanced Code Editor** 3.3.0 includes the following change.

=== Advanced Code Editor did not preserve changes to _font size_ or _dark/light mode_ state.
//#TINY-9749

In previous versions of {productname}, the _font size_ and _dark/light mode_ states within the Advanced Code Editor were not preserved after closing the dialog.

When the Advanced Code Editor was closed, these states reverted to their default values.

In {productname} 6.5, _font size_ and _dark/light mode_ states are preserved when the Advanced Code Editor is closed.

NOTE: This issue only affected the Advanced Coded Editor when it was opened in a dialog.

For information on the **Advanced Code Editor** plugin, see: xref:advcode.adoc[Advanced Code Editor].
```
Edits.

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 4]
```diff
@@ -30,6 +30,25 @@
...
+=== Advanced Code Editor 3.3.0
+
+The {productname} 6.5 release includes an accompanying release of the **Advanced Code Editor** premium plugin.
+
+**Advanced Code Editor** 3.3.0 includes the following change.
+
+=== Advanced Code Editor did not preserve changes to _font size_ or _dark/light mode_ state.
+//#TINY-9749
+
+In previous versions of {productname}, the _font size_ and _dark/light mode_ states within the Advanced Code Editor were not preserved after closing the dialog.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:42]:
```suggestion
In previous versions of the Advanced Code Editor, the _font size_ and _dark/light mode_ states within the Advanced Code Editor were not preserved after closing the dialog.
```

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 5]
```diff
@@ -30,6 +30,25 @@
...
+=== Advanced Code Editor did not preserve changes to _font size_ or _dark/light mode_ state.
+//#TINY-9749
+
+In previous versions of the Advanced Code Editor, the _font size_ and _dark/light mode_ states within the Advanced Code Editor were not preserved after closing the dialog.
+
+When the Advanced Code Editor was closed, these states reverted to their default values.
+
+In {productname} 6.5, _font size_ and _dark/light mode_ states are preserved when the Advanced Code Editor is closed.
+
+NOTE: This issue only affected the Advanced Coded Editor when it was opened in a dialog.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:48]:
```suggestion
In previous versions of the **Advanced Code Editor**, the _font size_ and _dark/light mode_ states were not preserved after closing the dialog.

When the **Advanced Code Editor** was closed, these states reverted to their default values.

In **Advanced Code Editor** 3.3.0, _font size_ and _dark/light mode_ states are preserved when the editor is closed.

NOTE: This issue only affected the **Advanced Coded Editor** when it was opened in a dialog.
```

**Review by brian-forte — APPROVED** [review]

**Pull request merged**